README.md

```diff
@@ -377,7 +373,7 @@ Negated `_.isNil`
 Returns true if the input has a `length` property > 1, such as arrays, strings, or custom objects with a lenth property
 
 ### append
-A curried, flipped `add`
+A curried, flipped `+`. The flipping matters for strings, e.g. `F.append('a')('b') -> 'ba'`
```

Comment:
we should mention that is a curried, flipped `_.add` and not the `+` operator as there may small differences